scripts/common.Makefile

```diff
@@ -253,13 +260,13 @@ venv: $(REPO_BASE_DIR)/.venv/bin/activate ## Creates a python virtual environmen
 ifeq ($(shell test -f j2cli_customization.py && echo -n yes),yes)
 
 define jinja
-	$(REPO_BASE_DIR)/.venv/bin/j2 --format=env $(1) .env -o $(2) --customize j2cli_customization.py
+	$(REPO_BASE_DIR)/.venv/bin/j2 --format=env $(1) $(2) -o $(3) --customize j2cli_customization.py
```

Comment:
For me my original proposal would fragment the code less than your suggestions, I think it is fine for all callers to explicitly provide the path/name to the .env-file. I would prefer not to have more special jinja functions, at least when it can be avoided. And in this case it is just about being able to specify the name/path of the `.env` file explicitly.

In the end this is a question of paradigms, I guess... I personally subscribe to "Don't use defaults, but validate and fail if expected input is not provided"

Let's also sync in person.